src/Servers/Connections.Abstractions/src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,19 @@
 #nullable enable
+Microsoft.AspNetCore.Connections.TlsConnectionCallbackContext
```

Comment:
I added some types to Connections.Abstractions but they use SslServerAuthenticationOptions which isn't available in .NET Standard or .NET Framework. I put an ifdef around them for .NET 7 or later only, but I get:

> src/Servers/Connections.Abstractions/src/PublicAPI.Unshipped.txt(2,1): error RS0017: (NETCORE_ENGINEERING_TELEMETRY=Build) Symbol 'Microsoft.AspNetCore.Connections.TlsConnectionCallbackContext' is part of the declared API, but is either not public or could not be found

How can I fix this?

Reply:
cc @dougbu @wtgodbe

Reply:
https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md#conditional-api-differences

Reply:
See also what @davidfowl added in the Microsoft.AspNetCore.SignalR.Common.csproj project and its src/SignalR/common/SignalR.Common/src/PublicAPI folder